resources/2024-10-dapr-bindings.md

```diff
@@ -0,0 +1,276 @@
+# Title
+
+* **Author**: Lucas Peirone (@SoTrx)
+
+## Overview
+
+Currently, Radius support for Dapr is incomplete. This design document aims to add support for another Dapr Building Block: Bindings.
```

Comment:
The first sentence may raise more questions rather than help provide an overview for the design. Adding Dapr Bindings will be great but will not complete support for Dapr. Maybe we avoid the first sentence and add more information on how adding Dapr Bindings will ,say, help resources interact with external systems in a decoupled manner.